sdk/storage/Azure.Storage.Common/tests/Shared/TestConfigurationsTemplate.xml

```diff
@@ -13,181 +13,168 @@
 
   <TenantConfigurations>
     <!-- A storage account with an Encryption Scope, Blob Versioning, and Last Access Time enabled -->
-    <!-- Replace [ACCOUNT] and [ACCOUNT-KEY] -->
```

Comment:
NIT why are the brackets removed?

Reply:
My understanding is that `[` and `]` are invalid XML characters.  This means that before this change, you had fill in all the tests accounts, even if you only needed to run one type of test.